build_library/build_image_util.sh

```diff
@@ -697,7 +697,25 @@ EOF
 
   # Build the selinux policy
   if pkg_use_enabled coreos-base/coreos selinux; then
```

Comment:
Please move this block before `write_contents` and `sudo fstrim`, otherwise the image content list is not correct and the fstrim doesn't have the full effect to reduce the image size to a minimum.